frontend/packages/kubevirt-plugin/src/yaml-templates.ts

```diff
@@ -1,10 +1,12 @@
 import { Map as ImmutableMap } from 'immutable';
 
+import { TemplateModel } from '@console/internal/models';
 import { VirtualMachineModel } from './models';
 
-export const yamlTemplates = ImmutableMap().setIn(
-  [VirtualMachineModel, 'default'],
-  `
+export const yamlTemplates = ImmutableMap()
```

Comment:
can we move this file to a special folder to not to polute the `src`?

Reply:
@suomiy this yaml const is used by vms, vm-templates and in the future, all other resources.

I agree about moving this file, because moving it impacts not only this PR but also existing `vms` resources and the future resources we will import from web-ui, it should be done in a separate PR and inform all people working on the that we intend to move this file.

Reply:
ok it can be done later if you think this is a hassle (even though the conflicts should be resolvable in each PR)

Reply:
+1 on that.

I'd suggest to create `src/models` and `git mv src/models.ts src/models/index.ts`.

YAML templates are inherently related to models, so I'd add `src/models/yaml-templates.ts`.